docs/advanced-features/athena.md

```diff
@@ -7,6 +7,129 @@ You can therefore query package metadata wth SQL engines like AWS Athena.
 Users can write SQL queries to select packages (or files from within packages)
 using predicates based on package or object-level metadata.
 
+## Prerequisites: Athena setup
+
+To get enable Athena, you need to set up a role with policy allowing Athena.
+Steps required to do this:
+
+1. Create Athena policy. Go to console.aws.amazon.com/iam, and create new policy with this JSON.
+```
+{
```

Comment:
Is this `arn:aws:iam::aws:policy/AmazonAthenaFullAccess`? If so I don't think there is sense to copy it here because it's AWS managed and already exists in every account. Also I think it provides more permissions than we need, so I think it either should be restricted or we should say it might be too permissive.